packages/web3-test/package.json

```diff
@@ -22,11 +16,25 @@
     "url": "https://github.com/alephium/alephium-web3/issues"
   },
   "author": "Alephium dev <dev@alephium.org>",
+  "main": "dist/_cjs/index.js",
+  "module": "dist/_esm/index.js",
+  "browser": "dist/alephium-web3-wallet.min.js",
```

Comment:
This is the package.json of the web3-test package but web3-wallet is referenced in this line. Is this correct?

Reply:
It's removed since there is not build for browser for this package.